ang/civicase/ActivityFeed.js

```diff
@@ -432,25 +425,29 @@
        */
       function affixActivityDetailsPanel () {
         $timeout(function () {
-          $activityDetailsPanel = $element.find('.panel');
+          $activityDetailsPanel = $element.find('.civicase__activity-panel');
           $filter = $('.civicase__activity-filter');
           $feedListContainer = $('.civicase__activity-feed__list-container');
           $tabs = $('.civicase__dashboard').length > 0 ? $('.civicase__dashboard__tab-container ul.nav') : $('.civicase__case-body_tab');
           $toolbarDrawer = $('#toolbar');
 
           $activityDetailsPanel.affix({
             offset: {
-              top: $element.find('.panel').offset().top - ($toolbarDrawer.height() + $tabs.height() + $filter.height()),
+              top: $activityDetailsPanel.offset().top - ($toolbarDrawer.height() + $tabs.height() + $filter.height()),
               bottom: $($document).height() - ($feedListContainer.offset().top + $feedListContainer.height())
             }
-          }).on('affixed.bs.affix', function () {
+          });
+
+          $activityDetailsPanel.on('affixed.bs.affix', function () {
             $activityDetailsPanel
               .css('top', ($toolbarDrawer.height() + $tabs.height() + $filter.height()))
-              .css('padding-top', 32);
+              .css('padding-top', 32)
+              .width($element.width());
           }).on('affixed-top.bs.affix', function () {
             $activityDetailsPanel
               .css('top', 'auto')
-              .css('padding-top', 0);
+              .css('padding-top', 0)
+              .css('width', 'auto');
```

Comment:
please explain why we need `width: auto` set via JS. In the PR you wrote 

> Previously a css approach was used for the same, but it was producing inconsistent results.

but that doesn't say much. Why the width of the elements should be affected at all?

Reply:
but then why don't we simply toggle a css class that will set the correct width value? The value of `width` doesn't need to be handled by JS

Reply:
Using CSS to define the width is perfectly fine. I tested the behaviour and it's exactly the same.